CITATION.cff

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# This CITATION.cff file was initially generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: alchemlyb
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - email: dotsdl@gmail.com
+    given-names: David L.
+    family-names: Dotson
+    orcid: 'https://orcid.org/0000-0001-5879-2942'
+    affiliation: Datryllic LLC
+  - given-names: Ian M.
+    family-names: Kenney
+    orcid: 'https://orcid.org/0000-0002-9749-8866'
+    affiliation: Kenney Consulting, LLC    
+  - given-names: Oliver
+    family-names: Beckstein
+    email: obeckste@asu.edu
+    affiliation: Arizona State University
+    orcid: 'https://orcid.org/0000-0003-1340-0831'
+  - given-names: Shuai
+    family-names: Liu
+    orcid: 'https://orcid.org/0000-0002-8632-633X'
+    affiliation: Silicon Therapeutics LLC
+  - given-names: Travis
+    family-names: Jensen
+  - given-names: Bryce
+    family-names: Allen
+    orcid: 'https://orcid.org/0000-0002-0804-8127'
+    affiliation: Differentiated Therapeutics
+  - given-names: Dominik
+    family-names: Wille
+    affiliation: Freie Universität Berlin
+  - given-names: Victoria
+    family-names: Lim
+    orcid: 'https://orcid.org/0000-0003-4030-9312'
+    affiliation: University of California Irvine
+  - given-names: Hyungro
+    family-names: Lee
+    orcid: 'https://orcid.org/0000-0002-4221-7094'
+    affiliation: Pacific Northwest National Laboratory
+  - given-names: Mohammad Soroush
+    family-names: Barhaghi
+    orcid: 'https://orcid.org/0000-0001-8226-7347'
+    affiliation: Wayne State University
+  - given-names: Zhiyi
+    family-names: Wu
+    affiliation: Exscientia plc
+    orcid: 'https://orcid.org/0000-0002-7615-7851'
+    email: william@zhiyiwu.me
```

Comment:
@xiki-tempula are you ok with this information?

Reply:
Yes, I'm happy.

Reply:
thanks